VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-details/hearing-details.component.html

```diff
@@ -0,0 +1,27 @@
+<table class="govuk-table vh-fullwidth" [attr.aria-label]="'judge-waiting-room.details-of-the-hearing' | translate">
+  <thead class="govuk-table__head no-mass">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">{{ 'judge-waiting-room.hearing' | translate }}</th>
+      <th scope="col" class="govuk-table__header">{{ 'judge-waiting-room.date' | translate }}</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell hearing-details-col1-width">
+        <strong><app-truncatable-text [text]="conference.caseName" [maxLimit]="40"></app-truncatable-text></strong>
+        ({{ 'case-type.' + stringToTranslateId(conference.caseType) | translate }})
+        <br />
+        <span>{{ 'judge-waiting-room.case-number' | translate }}</span
```

Comment:
a bit of house keeping to create a new section 'waiting-room' so we can remove the room specific translations. that way we can track what translations are still being used and remove what isn't

Reply:
should be consolidated now. I've removed the sections from the translations `(judge-waiting-room`, `joh-waiting-room`, `participant-waiting-room`) and replaced them with a single consolidated `waiting-room` section